requirements.txt

```diff
@@ -1,3 +1,4 @@
 .
 black==21.9b0
+glom==20.11.0
```

Comment:
I need this only for testing. Do we have a way to specify that a library need not be imported for production/deploy?

Reply:
Yes, this is the place for that. Runtime dependencies are defined in `setup.py`